tests/test_prettytable.py

```diff
@@ -2470,6 +2470,68 @@ def test_min_table_width(
                 ]
 
 
+class TestBreakOnHyphens:
+    row = [
+        "bluedevil breeze breeze-gtk eos-bash-shared glib2 kactivitymanagerd kde-cli-tools kde-gtk-config kdecoration kdeplasma-addons khotkeys kinfocenter kmenuedit kpipewire kscreen kscreenlocker ksystemstats kwallet-pam kwin layer-shell-qt lib32-glib2 libarchive libkscreen libksysguard libwebp milou oxygen-sounds plasma-desktop plasma-disks plasma-integration plasma-nm plasma-pa plasma-workspace polkit-kde-agent powerdevil python-gobject qt5-webengine sddm-kcm sqlite systemsettings vivaldi"  # noqa: E501
```

Comment:
Let's make it a bit smaller:
```suggestion
        "bluedevil breeze breeze-gtk eos-bash-shared glib2 "
        "kactivitymanagerd kde-cli-tools kde-gtk-config kdecoration"
```